cmd/kured/os_specific_behaviors.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/google/shlex"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type osSpecificBehaviors interface {
```

Comment:
I'm open to suggestions for better names for any of the objects in this file :-/